api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -122,21 +122,21 @@ public IActionResult FindWithDatabase()
 
     /// <summary>
     /// Find a page of content for the specified query filter.
+    /// TODO: The model stored in Elasticsearch is a little confusing based on the controller using it.  Need to clean up.
     /// </summary>
     /// <returns></returns>
     [HttpGet]
     [Produces(MediaTypeNames.Application.Json)]
-    [ProducesResponseType(typeof(IPaged<ContentModel>), (int)HttpStatusCode.OK)]
+    [ProducesResponseType(typeof(IPaged<Services.Models.Content.ContentModel>), (int)HttpStatusCode.OK)]
```

Comment:
We index content into Elasticsearch with the Indexing Service.  This service uses the above model.  Which means we generally should be using the same model to extract content from Elasticsearch.